test/acceptance/features/steps/steps.py

```diff
@@ -557,3 +562,13 @@ def update_binding_secret(context):
     output = openshift.apply(yaml.dump(secret_yaml), context.namespace.name)
     result = re.search(rf'.*{context.sb_secret}.*(created|unchanged|configured)', output)
     assert result is not None, f"Unable to apply YAML for binding secret '{context.sb_secret}': {output}"
+
+
+@given(u'CustomResourceDefinition backends.stable.example.com is available')
+def backends_crd(context):
+    openshift = Openshift()
+    if "namespace" in context:
+        ns = context.namespace.name
+    else:
+        ns = None
+    _ = openshift.apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/backend_crd.yaml"), namespace=ns)
```

Comment:
please avoid DRY - this looks very similar to `operator_manifest_installed` method, let's generalize it if needed, and add additional step definition to it.

Reply:
Done.